.github/workflows/test.yml

```diff
@@ -46,8 +46,8 @@ jobs:
       fail-fast: false
       matrix:
         jdk: [11, 17, 21, 22]
-        maven: [3.9.6]
-        image: [java]
+        maven: [3.9.9]
+        image: [java, conda-java-jni-cdata]
```

Comment:
could we add something on the Job name about whether is JNI or not? Otherwise it currently appears as two jobs with the same name and could be difficult to find if you are looking for a specific job.

Reply:
Updated.